README.md

```diff
@@ -27,19 +27,58 @@ If you have a capable (Nvidia, at least 8GB of VRAM) GPU, it is highly recommend
 
 ## Training a model
 
-Open a terminal, change to the directory of the DeepSpeech checkout and run
+The central (Python) script is `DeepSpeech.py` in the project's root directory:
+
+```bash
+$ ./DeepSpeech.py --help
+```
+
+To get the output of this in a slightly better formatted way, you can also look up the option definitions top of `DeepSpeech.py` (line 40++).
+For executing pre-configured training scenarios, there is a collection of convenience scripts in the `bin` folder. Most of them are named after the corpora they are configured for. Keep in mind that the other speech corpora are *very large*, on the order of tens of gigabytes, and some aren't free. Downloading and preprocessing them can take a very long time, and training on them without a fast GPU (GTX 10 series recommended) takes even longer. If you experience GPU OOM errors while training, try reducing `batch_size`.
+
+As a simple first example you can open a terminal, change to the directory of the DeepSpeech checkout and run:
+
 ```bash
 DeepSpeech$ ./bin/run-ldc93s1.sh
 ```
-By default, the code will train on a small sample dataset called LDC93S1, which can be overfitted on a GPU in a few minutes for demonstration purposes. From here, you can alter any variables with regards to what dataset is used, how many training iterations are run and the default values of the network parameters. Then, just run the script to train the modified network.
 
-You can also use other utility scripts in `bin/` to train on different data sets, but keep in mind that the other speech corpora are *very large*, on the order of tens of gigabytes, and some aren't free. Downloading and preprocessing them can take a very long time, and training on them without a fast GPU (GTX 10 series recommended) takes even longer. If you experience GPU OOM errors while training, try reducing `batch_size`.
+This script will train on a small sample dataset called LDC93S1, which can be overfitted on a GPU in a few minutes for demonstration purposes. From here, you can alter any variables with regards to what dataset is used, how many training iterations are run and the default values of the network parameters.
+Feel also free to pass additional (or overriding) `DeepSpeech.py` parameters to these scripts.
+Then, just run the script to train the modified network.
+
+## Checkpointing
+
+During training of a model so called checkpoints will get stored on disk. This takes place at a configurable time interval. Files are also rotated. The purpose of checkpoints is to allow interruption (also in case of some unexpected failure) and later continuation of training without loosing hours of training time. Resuming from checkpoints happens automatically by just (re)starting training with the same `--checkpoint_dir` of the former run.
```

Comment:
The checkpoint files will be rotated? (From the text it isn't clear that the checkpoint files will be rotated.)